src/static/js/header.js

```diff
@@ -324,6 +324,12 @@ header = {
             if (asm.locale != "en_GB") {
                 $(".taggb").hide();
             }
+            if (config.bool("DisableEvents")){
+                $(".event").hide();
+            }
+            if (!common.has_permission("ae") && !common.has_permission("ve")){
+                $(".event").hide();
+            }
```

Comment:
You don't need to do this - that's why the first element of menu_structure items is a permission.